cratedb_toolkit/datasets/README.md

```diff
@@ -0,0 +1,12 @@
+# CrateDB Datasets API
+
+Provide access to datasets, to be easily consumed by tutorials
+and/or production applications.
+
+## Synopsis
+```python
+from cratedb_toolkit.datasets import load_dataset
+
+dataset = load_dataset("tutorial/weather-basic")
+dataset.to_cratedb(dburi="crate://crate@localhost/", table="weather_data")
+```
```

Comment:
Would be nice to optionally use a `drop=True` option.

```suggestion
```python
from cratedb_toolkit.datasets import load_dataset

dataset = load_dataset("tutorial/weather-basic")
dataset.to_cratedb(dburi="crate://crate@localhost/", table="weather_data", drop=True)
```
```

Reply:
The function signature is now:

```python
def to_cratedb(
    self,
    dburi: str,
    table: str,
    if_exists: t.Literal["append", "noop", "replace"] = "noop",
    drop: bool = False,
):
    """
    Load dataset into CrateDB, using SQL.
    """
    pass
```